admin_guide/configuring_aws.adoc

```diff
@@ -0,0 +1,91 @@
+= Configuring for AWS
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+OpenShift can be configured to access an link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/concepts.html[AWS EC2 infrastructure], 
+including link:persistent_storage/persistent_storage_aws.html[using AWS volumes as persistent storage] for application data.
+After AWS is configured properly, some additional configurations will need to be completed on the OpenShift hosts.
+
+
+== Configuring AWS Variables
+To set the required AWS variables, create a *_/etc/aws/aws.conf_* file with
+the following contents on all of your OpenShift hosts, both masters and nodes:
+
+====
+----
+[Global]  
+Zone = us-east-1c  <1>
+----
+
+<1> This is the Availability Zone of your AWS Instance and where your EBS Volume resides - this information is obtained from the AWS Managment Console
+====
+
+
+
+== Configuring Masters
+
+Edit or
+link:master_node_configuration.html#creating-new-configuration-files[create] the
+master configuration file on all masters
+(*_/etc/openshift/master/master-config.yaml_* by default) and update the
+contents of the `*apiServerArguments*` and `*controllerArguments*` sections:
+
+====
+[source,yaml]
+----
+kubernetesMasterConfig:
+  ...
+  apiServerArguments:
+    cloud-provider:
+      - "aws"
+    cloud-config:
+      - "/etc/aws/aws.conf"
+  controllerArguments:
+    cloud-provider:
+      - "aws"
+    cloud-config:
+      - "/etc/aws/aws.conf"
+----
+====
+
+== Configuring Nodes
+
+Edit or
+link:master_node_configuration.html#creating-new-configuration-files[create] the
+node configuration file on all nodes
+(*_/etc/openshift/node-<hostname>/node-config.yaml_* by default) and update the
+contents of the `*kubeletArguments*` section:
+
+====
+[source,yaml]
+----
+kubeletArguments:
+  cloud-provider:
+    - "aws"
+  cloud-config:
+    - "/etc/aws/aws.conf"
+----
+====
+
+
+
+== Exporting Key Value Access Pairs
+
+(Re-)start OpenShift services on the master and all nodes and make sure they run with these environment variables:
```

Comment:
s/(Re-)start/Start or restart/

s/they/the hosts/

^ This accurate? Or did "they" refer to the services.